src/constants.ts

```diff
@@ -876,8 +876,11 @@ export type SupportedAIModels =
 	| `openai:${AIModels<'openai'>}`
 	| 'vscode';
 
-const supportedCloudIntegrationIds = ['github', 'jira'];
-export type SupportedCloudIntegrationIds = HostingIntegrationId.GitHub | IssueIntegrationId.Jira;
+const supportedCloudIntegrationIds = ['github', 'gitlab', 'jira'];
```

Comment:
Can you use the enums similar to the union type after it?

```typescript
const supportedCloudIntegrationIds = [
	HostingIntegrationId.GitHub,
	HostingIntegrationId.GitLab,
	IssueIntegrationId.Jira,
];
```

Reply:
fixed